### PR TITLE
[go/en] Add Golang University playlist links

### PR DESCRIPTION
**Pull request by bitfield**
- [X] I solemnly swear that this is all original content of which I am the original author
- [X] Pull request title is prepended with `[language/lang-code]`
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [X] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: go.html.markdown [hunk 1]
```diff
@@ -472,4 +473,9 @@ documentation](http://golang.org/pkg/) and the source code comes up!
 
 Another great resource to learn Go is [Go by example](https://gobyexample.com/).
 
+There are many excellent conference talks and video tutorials on Go available on YouTube, and Go trainer [John Arundel](https://bitfieldconsulting.com/golang) has put together three playlists of the very best, tailored for beginners, intermediate, and advanced Gophers respectively:
+* [Golang University 101](https://www.youtube.com/playlist?list=PLEcwzBXTPUE9V1o8mZdC9tNnRZaTgI-1P) introduces fundamental Go concepts and shows you how to use the Go tools to create and manage Go code
+* [Golang University 201](https://www.youtube.com/playlist?list=PLEcwzBXTPUE_5m_JaMXmGEFgduH8EsuTs) steps it up a notch, explaining important techniques like testing, web services, and APIs
+* [Golang University 301](https://www.youtube.com/playlist?list=PLEcwzBXTPUE_5m_JaMXmGEFgduH8EsuTs) dives into more advanced topics like the Go scheduler, implementation of maps and channels, and optimisation techniques
```
Comment (mxschumacher) [inline go.html.markdown]: the links for 201 and 301 are identical

File: go.html.markdown [hunk 2]
```diff
@@ -472,4 +473,9 @@ documentation](http://golang.org/pkg/) and the source code comes up!
 
 Another great resource to learn Go is [Go by example](https://gobyexample.com/).
 
+There are many excellent conference talks and video tutorials on Go available on YouTube, and Go trainer [John Arundel](https://bitfieldconsulting.com/golang) has put together three playlists of the very best, tailored for beginners, intermediate, and advanced Gophers respectively:
```
Comment (mxschumacher) [inline go.html.markdown]: The video playlists are fine for me, but I think the link to the commercial training of the author need to be removed. What do you think @divayprakash ?

File: go.html.markdown (hunk 1 above)
Comment (bitfield) [reply go.html.markdown]:
```suggestion
* [Golang University 301](https://www.youtube.com/watch?v=YHRO5WQGh0k&list=PLEcwzBXTPUE8KvXRFmmfPEUmKoy9LfmAf) dives into more advanced topics like the Go scheduler, implementation of maps and channels, and optimisation techniques
```

File: go.html.markdown (hunk 2 above)
Comment (divayprakash) [reply go.html.markdown]: I agree completely. Also, the author of the PR has put in the same link in the YAML header also, which would need to be removed too.

Comment (mxschumacher) [reply go.html.markdown]: yep, makes sense.

@bitfield

Comment (bitfield) [reply go.html.markdown]: > the author of the PR has put in the same link in the YAML header also

Sorry about that, I saw that previous contributors had included web links too, and assumed that it was OK for me to do that. Is it not?

Comment (mxschumacher) [reply go.html.markdown]: The link to your Github account is okay, the link to your commercial training offers is not. Learn X in Y minutes is not there to create traffic to your website.

As long as the sentence "and Go trainer John Arundel of Bitfield Consulting" and the link "https://bitfieldconsulting.com/golang" is in the PR, I will not merge this.

People who click on the playlist link will see the name of your company, so will people who look at your Github profile.

Comment (bitfield) [reply go.html.markdown]: That's perfectly fine! I wasn't arguing with you, just asking for information. As you'll appreciate, it's not always completely clear for new contributors to a project what the rules are, especially if they're not actually written down anywhere. Thanks for the clarification, and I've made the changes you asked for.

Comment (mxschumacher) [reply go.html.markdown]: Thanks for making the changes, I'll merge this now.

Comment (bitfield) [reply go.html.markdown]: Much appreciated, thank you!

**Pull request merged**